Code/GraphMol/Fingerprints/FingerprintGenerator.cpp

```diff
@@ -314,10 +389,32 @@ FingerprintGenerator<OutputType>::getFingerprint(
     const ROMol &mol, FingerprintFuncArguments &args) const {
   std::uint32_t effectiveSize = dp_fingerprintArguments->d_fpSize;
   if (dp_fingerprintArguments->df_countSimulation) {
-    // effective size needs to be smaller than result size to compansate for
+    // effective size needs to be smaller than result size to compensate for
     // count simulation
     effectiveSize /= dp_fingerprintArguments->d_countBounds.size();
   }
+
+  AdditionalOutput countSimulationOutput;
```

Comment:
Not a big deal, but we could make this code not duplicated in the two functions.

Reply:
Yep, good idea. I did that.